### PR TITLE
UI styling

### PR DESCRIPTION
**Pull request by samwel-mongare**
This is a pair-programming Project for Microverse created with HTML and Javascript by Kalolo Chola Lemba and Sam Mongare. In this milestone we did the following:

- Added styles to the user interface using CSS from the styles.css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,23 +5,37 @@
...
+    <h1>All awesome books</h1>
+    <table>
+        <thead>
+            <tr>
+            <th>Title</th>
+            <th>Author</th>
+            <th></th>
+            </tr>
+        </thead>
+        <tbody id="book-list"></tbody>
```
Comment (SajjadAhmad14) [inline index.html]: Please follow the wireframe for the books list.
Yours:
Yours
Wireframe:
o

File: index.html [hunk 2]
```diff
@@ -5,23 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```
Comment (SajjadAhmad14) [inline index.html:5]: Please add all unnecessary files and folders like `vscode` to `gitignore`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Sam and Kaylo, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 2 above)
Comment (KayLemba) [reply index.html:5]: Done

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi Team🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**